drivers/usb/device/Kconfig

```diff
@@ -21,6 +21,22 @@ config USB_DW_IRQ_PRI
 	help
 	USB interrupt priority.
 
+config USB_DC_STM32
+	bool
+	prompt "USB device controller driver for STM32 devices"
+	depends on HAS_STM32CUBE
+	depends on SOC_SERIES_STM32F4X
+	default n
+	help
+	STM32 family USB device controller Driver.
+
+config USB_DC_STM32_IRQ_PRI
```

Comment:
lets add USB to DT and get interrupt & interrupt priority from there instead.

Reply:
Is it possible to do it in a following PR?

Reply:
I think this request some DT parsing enhancement, I would prefer make this in a separate PR so that we could apply this for both USB_DC_STM and USB_DW device controller drivers.